### ISSUE TITLE
`exclude` does not work in 0.15.1

### ISSUE THREAD

**Issue by printercu** [sds/slim-lint#77]
Hi!

Here is my config:

```yml
linters:
  LineLength:
    max: 100
    exclude:
    - app/views/site/abouts/terms_and_conditions.html.slim
```

It ignores the file in 0.15.0 but not in 0.15.1. Also all paths for warnings are shown expanded instead of relative to project root. Seems like https://github.com/sds/slim-lint/commit/ef9a07f5c063bf7d766e770483b1f49c44d4bc87 caused this issue.

**Comment by sds** [comment]
Thanks for the bug report, @printercu. I would appreciate a pull request fixing this, as I won't have bandwidth to get to this for some time. Thanks!
